### ISSUE TITLE
Now watches .jsx files by default

### ISSUE THREAD

**Issue by km-tr** [jedwards1211/flow-watch#9]
It corresponds to jsx in v1.1.2, but does not it have to specify extension?

`-e js,jsx`

**Comment by jedwards1211** [comment]
I just corrected the README: if you run `flow-watch` without any arguments, then it's the same as if you had run it with
```
flow-watch --ignore node_modules/ --watch *.js --watch *.jsx --watch *.js.flow --watch .flowconfig
```
But if you do provide any arguments, then those defaults no longer apply, so
```
flow-watch --watch *.js
```
would not watch `.jsx`, `.js.flow` files, or `.flowconfig`, and would not ignore `node_modules`.

**Comment by km-tr** [comment]
I think nodemon does not correspond to jsx extension by default.

https://github.com/remy/nodemon#specifying-extension-watch-list

**Comment by jedwards1211** [comment]
Oh, I think you have a good point.  I need to investigate this tomorrow.  So when you change a `.jsx` file, `flow-watch` isn't rerunning flow?

**Comment by km-tr** [comment]
Yes.

**Comment by jedwards1211** [comment]
@km-tr sorry for the long delay on this.  I just published a fix in version 1.1.3.
